rails_boot/benchmarks/driver.rb

```diff
@@ -0,0 +1,156 @@
+#
+# Bundler Benchmark driver
+#
+require 'net/http'
+require 'json'
+require 'pathname'
+require 'optparse'
+require 'digest'
+require 'benchmark'
+require 'benchmark/ips'
+require 'fileutils'
+
+RAW_URL = 'https://raw.githubusercontent.com/ruby-bench/ruby-bench-suite/master/rails/benchmarks/'
+
+class BenchmarkDriver
+  def self.benchmark(options)
+    self.new(options).run
+  end
+
+  def initialize(options)
+    @repeat_count = options[:repeat_count]
+    @local = options[:local]
+    @versions = options[:versions]
+  end
+
+  def run
+    versions_to_run = @versions || %w(4.2.8 5.0.0.1 5.0.1)
+    versions_to_run.each do |version|
+      run_single(version)
+    end
+  end
+
+  private
+
+  def files
+    Pathname.glob("#{File.expand_path(File.dirname(__FILE__))}/bm_*")
+  end
+
+   def run_single(version)
+    system('gem uninstall rails --all --force')
+    system('gem uninstall activesupport --all --force')
+    system("gem install rails:#{version} --no-doc")
+    system("rails _#{version}_ new my_app_#{version}")
+
+    scripts = {
+      '4' => [
+        'bin/rake environment'
+      ],
+      '5' => [
+        'bin/rails test',
+        'bin/rake environment'
+      ]
+    }
+
+    Dir.chdir("my_app_#{version}") do
+      scripts[version[0]].each do |script|
+        output = measure(script, version)
+        return unless output
+
+        request = Net::HTTP::Post.new('/benchmark_runs')
+        request.basic_auth(ENV["API_NAME"], ENV["API_PASSWORD"])
+
+        submit = {
+          'benchmark_type[category]' => output["label"],
+          'benchmark_type[script_url]' => "#{RAW_URL}#{version}",
```

Comment:
As jerry pointed out, we dont have a script url actually, this should be removed.